ignition/api/exceptions.py

```diff
@@ -19,7 +19,7 @@ def __init__(self, exception_type, handler):
         if not callable(handler):
             raise ValueError('handler must be a callable type')
         handler_args = inspect.getfullargspec(handler).args
-        valid_handler = (len(handler_args) is 1) or (len(handler_args) is 2 and handler_args[0] is 'self')
+        valid_handler = (len(handler_args) == 1) or (len(handler_args) == 2 and handler_args[0] == 'self')
```

Comment:
has this been changed because of the python uplift or is it just something  you noticed needed fixing?

Reply:
Yeah, the old way worked but Python prints warnings for this now.